patterns/ecr-pull-through-cache/addons.tf

```diff
@@ -67,11 +109,28 @@ module "gatekeeper" {
   chart            = "gatekeeper"
   chart_version    = "3.16.3"
   repository       = "https://open-policy-agent.github.io/gatekeeper/charts"
-  values = [
-    templatefile("${path.module}/values/gatekeeper.yaml", {
-      ecr_account_id = local.ecr_account_id
-      ecr_region     = local.ecr_region
-  })]
+  values = [<<-EOT
+    image:
+      repository: ${local.ecr_url}/docker-hub/openpolicyagent/gatekeeper
+      crdRepository: ${local.ecr_url}/docker-hub/openpolicyagent/gatekeeper-crds
+    postUpgrade:
+      labelNamespace:
+        image:
+          repository: ${local.ecr_url}/docker-hub/openpolicyagent/gatekeeper-crds
+      postInstall:
+        labelNamespace:
+          image:
+            repository: ${local.ecr_url}/docker-hub/openpolicyagent/gatekeeper-crds
+      probeWebhook:
+        image:
+          repository: ${local.ecr_url}/docker-hub/curlimages/curl
+      preUninstall:
+        deleteWebhookConfigurations:
+          image:
+            repository: ${local.ecr_url}/docker-hub/openpolicyagent/gatekeeper-crds
+
+  EOT
+  ]
 
-  depends_on = [time_sleep.addons_wait_60_seconds]
+  depends_on = [module.eks_blueprints_addons]
```

Comment:
why does it need to depend on? should we just add this into the addons module above under the generic `helm_releases`?

Reply:
Fixing it, it should be  `depends_on = [module.eks.cluster_addons]`, since it requires VPC CNI to be up and running before start deploying, or it will break. 

I can also add on the -addons module.

Reply:
because we are using `wait = true`? we should not wait and let the controllers reconcile themselves (or don't deploy these with Terraform 😉 )